mscore/mixer.cpp

```diff
@@ -250,10 +253,19 @@ void Mixer::showEvent(QShowEvent* e)
       {
       enablePlay->showEvent(e);
       QWidget::showEvent(e);
-      activateWindow();
-      setFocus();
+      getAction("toggle-mixer")->setChecked(true);
       }
 
+//---------------------------------------------------------
+//   hideEvent
+//---------------------------------------------------------
+
+void Mixer::hideEvent(QHideEvent* e)
+{
+    QWidget::hideEvent(e);
+    getAction("toggle-mixer")->setChecked(false);
```

Comment:
6 spaces indent please...

Reply:
@Obliquely last review and we are ready to merge :)

Reply:
issue not fixed yet

Reply:
Sorry about that. Should be OK now.